CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 <!-- ### Changed -->
 
+- Changed hashing of backups to sha256 [#693](https://github.com/openkfw/TruBudget/issues/693)
```

Comment:
I'm not sure if a changelog entry is needed here as the changes are not really relevant for users